pkg/telemetry/sender_sio.go

```diff
@@ -0,0 +1,53 @@
+package telemetry
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/segmentio/analytics-go/v3"
+)
+
+const SegmentioEnvVariableName = "TESTKUBE_SEGMENTIO_KEY"
+
+var SegmentioKey = ""
+
+func SegmentioSender(client *http.Client, payload Payload) (out string, err error) {
```

Comment:
Where is `out` supplied with value?

Reply:
It's not :/ - Will refactor this Sender type after removing GA and debug adapter - added TODO item.